configs/apps/all/spack.yaml

```diff
@@ -7,6 +7,10 @@ spack:
   specs:
     - jedi-fv3-bundle-env
     - jedi-ufs-bundle-env
+    - jedi-um-bundle-env
+    # Do not add, this leads to duplicate packages being installed
+    #- jedi-tools-env
+    - nceplibs-bundle
```

Comment:
Might be worthwhile to figure out why duplicate packages are being installed. Probably a conflict somewhere.

Reply:
Do you know if your latest `py-setuptools` version change fixed the problem?

Reply:
I don't ... will try again after I get through all of the changes I have currently.